contracts/deploy/_11_new_mix_oracle.js

```diff
@@ -0,0 +1,196 @@
+const { getTxOpts } = require("../utils/tx");
+
+let totalDeployGasUsed = 0;
+const isTestMainnet = process.env.TEST_MAINNET_ORACLE;
+
+// Wait for 3 blocks confirmation on Mainnet/Rinkeby.
+const NUM_CONFIRMATIONS = isMainnet || isRinkeby ? 3 : 0;
+
+function log(msg, deployResult = null) {
+  if (isMainnet || isRinkeby || process.env.VERBOSE) {
+    if (deployResult) {
+      const gasUsed = Number(deployResult.receipt.gasUsed.toString());
+      totalDeployGasUsed += gasUsed;
+      msg += ` Address: ${deployResult.address} Gas Used: ${gasUsed}`;
+    }
+    console.log("INFO:", msg);
+  }
+}
+
+const newMixOracle = async ({ getNamedAccounts, deployments }) => {
+  let transaction;
+
+  const { deploy } = deployments;
+  const { governorAddr, deployerAddr } = await getNamedAccounts();
+  const oracleAddresses = await getOracleAddresses(deployments);
+  const assetAddresses = await getAssetAddresses(deployments);
+
+  console.log("Running 11_new_mix_oracle deployment...");
+
+  const sDeployer = ethers.provider.getSigner(deployerAddr);
+
+  // Deploy MixOracle.
+  // Note: the args to the MixOracle are as follow:
+  //  - for live the bounds are 1.3 - 0.7
+  //  - fot testing the bounds are 1.6 - 0.5
+  const MaxMinDrift = isMainnetOrRinkebyOrFork ? [13e7, 7e7] : [16e7, 5e7];
+  d = await deploy("MixOracle", {
+    from: deployerAddr,
+    args: MaxMinDrift,
+    ...(await getTxOpts()),
+  });
+  await ethers.provider.waitForTransaction(
+    d.receipt.transactionHash,
+    NUM_CONFIRMATIONS
+  );
+  log("Deployed MixOracle", d);
+
+
+  const chainlinkOracle = await ethers.getContract("ChainlinkOracle");
+
+  // Deploy the New OpenUniswap oracle.
+  d = await deploy("OpenUniswapOracle", {
+    from: deployerAddr,
+    args: [oracleAddresses.openOracle, assetAddresses.WETH],  // REMEMBER to update the oracle address
+    ...(await getTxOpts()),
+  });
+  await ethers.provider.waitForTransaction(
+    d.receipt.transactionHash,
+    NUM_CONFIRMATIONS
+  );
+  log("Deployed OpenUniswapOracle", d);
+
+  const uniswapOracle = await ethers.getContract("OpenUniswapOracle");
+  
+  t = await uniswapOracle
+    .connect(sDeployer)
+    .registerPair(oracleAddresses.uniswap.DAI_ETH, await getTxOpts());
+  await ethers.provider.waitForTransaction(t.hash, NUM_CONFIRMATIONS);
+  log("Registered uniswap pair DAI/ETH");
+
+  t = await uniswapOracle
+    .connect(sDeployer)
+    .registerPair(oracleAddresses.uniswap.USDC_ETH, await getTxOpts());
+  await ethers.provider.waitForTransaction(t.hash, NUM_CONFIRMATIONS);
+  log("Registered uniswap pair USDC/ETH");
+
+  t = await uniswapOracle
+    .connect(sDeployer)
+    .registerPair(oracleAddresses.uniswap.USDT_ETH, await getTxOpts());
+  await ethers.provider.waitForTransaction(t.hash, NUM_CONFIRMATIONS);
+  log("Registered uniswap pair USDT/ETH");
+
+  const mixOracle = await ethers.getContract("MixOracle");
+  if (isMainnetOrRinkebyOrFork) {
```

Comment:
Not sure we need this if / else statement given that we skip this migration if it's not Mainnet/Rinkeby/Fork.

I would just keep only the code that's in the if statement and not make it conditional. And get rid of all the code in the else statement. Basically just relying on the skip on live 194 to determine if this whole migration runs or not.